README.md

```diff
@@ -26,25 +26,23 @@ Add this line to your Rails application's Gemfile:
 
 ```ruby
 group :development do
-  # ...
-  gem "chusaku", require: false
-  # ...
+  gem "chusaku"
```

Comment:
This means that `chusaku` will now _always_ be required - while I don't think it'll be a huge problem, given how often `chusaku` probably won't be required I'd say it's still better to stick with the default of `require: false` here and encourage people to explicitly `require "chusaku"` when they need to (such as for loading the rake task).